Sources/MeiliSearch/Indexes.swift

```diff
@@ -58,6 +58,57 @@ struct Indexes {
 
     }
 
+    func getOrCreate(_ UID: String, _ completion: @escaping (Result<Index, Swift.Error>) -> Void) {
+
+        self.create(UID) { result in
+
+            switch result {
+
+            case .success(let index):
+                completion(.success(index))
+
+            case .failure(let error):
+                switch error {
+                case CreateError.indexAlreadyExists:
+                    self.get(UID, completion)
+                default:
+                    completion(.failure(error))
+                }
+
+            }
+
+        }
+
+    }
```

Comment:
@ppamorim No, I tell that because I see sometimes an empty line behind a scope close bracket. Like on these lines.

Reply:
From where this rule is coming from? `swiftlint` does not complain about it. If you want you can open an issue that we can fix it since this is all over the project.

Please check this out: https://google.github.io/swift/#braces

Reply:
@qdequele :)

Reply:
Thanks @ppamorim. It's not a problem. I'm sorry I slowed things down.